go.mod

```diff
@@ -19,6 +19,7 @@ require (
 	github.com/magiconair/properties v1.8.6
 	github.com/mitchellh/go-homedir v1.1.0
 	github.com/mitchellh/mapstructure v1.4.3
+	github.com/moby/term v0.0.0-20210619224110-3f7ff695adc6
```

Comment:
Any reason why not to use `golang.org/x/term` instead?

Reply:
No good reason. It was just the first package I saw while searching for a way to get the terminal size. I'll change it to `golang.org/x/term`

> Looks preetty!

It sure does

> Let's remove the generated.deepcopy files before merging, though.

Ok